force_wfmanager/central_pane/central_pane.py

```diff
@@ -23,16 +23,28 @@ class CentralPane(TraitsTaskPane):
     #: The plot view
     plot = Instance(Plot)
 
-    view = View(Tabbed(
+    #: Enable or disable the contained entities.
+    #: Used when the computation is running
+    enabled = Bool(True)
+
+    #: Button used to clear the model
+    clear_button = Button('Clear')
```

Comment:
I completely agree with you, I wasn't sure if this button would be useful anyway because the run button does the clear when starting a new computation.

Which makes me think that when the user clicks on the run button and there is already result data, we should pop up a confirmation dialog.